src/BookSettings.ts

```diff
@@ -116,12 +116,12 @@ export default class BookSettings {
             const viewOptions = this.bookViews.map(bookView =>
                 optionTemplate("", bookView.name, bookView.label)
             );
-            sections.push(sectionTemplate("View", viewOptions.join("")));
+            sections.push(sectionTemplate("", viewOptions.join("")));
```

Comment:
@aslagle maybe you dont want an empty string here... the compiler was complaining otherwise and I didnt want to muck about too much w/ template logic, so I did it this way to get the job done...